Assets/MRTK/Providers/Experimental/WindowsSceneUnderstanding/SceneUnderstandingObserverProfile.cs

```diff
@@ -0,0 +1,195 @@
+    /// </summary>
+    [CreateAssetMenu(menuName = "Mixed Reality Toolkit/Profiles/Scene Understanding Observer Profile", fileName = "SceneUnderstandingObserverProfile", order = (int)CreateProfileMenuItemIndices.SceneUnderstandingObserver)]
+    [MixedRealityServiceProfile(typeof(IMixedRealitySceneUnderstandingObserver))]
+    public class SceneUnderstandingObserverProfile : BaseSpatialAwarenessObserverProfile
+    {
+        #region IMixedRealityOnDemandObserver settings
+
+        [SerializeField]
+        [Tooltip("Observer will update once after initialization then require manual update thereafter.")]
+        private bool updateOnceOnLoad = false;
+        /// <summary>
+        /// Observer will update once after initialization then require manual update thereafter. Uses <see cref="FirstUpdateDelay"/> to determine when.
+        /// </summary>
+        public bool UpdateOnceOnLoad => updateOnceOnLoad;
+
+        [SerializeField]
+        [Tooltip("Should the observer update on an interval or on demand?")]
+        private bool autoUpdate = false;
+
+        /// <summary>
+        /// Indicates if the observer is to start immediately or wait for manual startup.
+        /// </summary>
+        /// <remarks>
+        /// When false, the observer will only update when UpdateOnDemand() is called.
+        /// </remarks>
+        public bool AutoUpdate => autoUpdate;
+
+        #endregion IMixedRealityOnDemandObserver settings
+
+        [PhysicsLayer]
+        [SerializeField]
+        [Tooltip("Physics layer on which to set understood planes.")]
+        private int defaultPhysicsLayer = 31;
+        /// <summary>
+        /// Physics layer on which to set understood planes
+        /// </summary>
+        public int DefaultPhysicsLayer => defaultPhysicsLayer;
+
+        [EnumFlags]
+        [SerializeField]
+        [Tooltip("Which plane types the observer should generate.")]
+        private SpatialAwarenessSurfaceTypes surfaceTypes = 
+            SpatialAwarenessSurfaceTypes.Floor | 
+            SpatialAwarenessSurfaceTypes.Ceiling | 
+            SpatialAwarenessSurfaceTypes.Wall |
+            SpatialAwarenessSurfaceTypes.Platform;
+
+        /// <summary>
+        /// Which plane types the observer should generate.
+        /// </summary>
+        public SpatialAwarenessSurfaceTypes SurfaceTypes => surfaceTypes;
+
+        [SerializeField]
+        [Tooltip("The number of planes to render per interval. Setting this too high can cause performance problems.")]
+        private int instantiationBatchRate = 1;
+        /// <summary>
+        /// Number of planes to spawn per frame. Used to throttle object creation for performance.
+        /// </summary>
+        public int InstantiationBatchRate => instantiationBatchRate;
+
+        [SerializeField]
+        [Tooltip("Material to use when displaying understood planes and meshes")]
+        private Material defaultMaterial = null;
```

Comment:
if these materials are null, does the feature still work? i do not see a visibility option similar to the mesh observer

Reply:
Yes, it still works but gameObjects instantiated will be pink.

Reply:
We should address that. Pink (missing material) objects in the scene are bad.